runtime/autoload/man.vim

```diff
@@ -1,6 +1,15 @@
-let s:man_tag_depth = 0
+" Ensure Vim is not recursively invoked (man-db does this)
+" by removing MANPAGER from the environment
+" More info here http://comments.gmane.org/gmane.editors.vim.devel/29085
+if &shell =~# 'fish$'
+  let s:man_cmd = 'man -P cat ^/dev/null '
+else
+  let s:man_cmd = 'man -P cat 2>/dev/null '
+endif
+
 let s:man_sect_arg = ''
 let s:man_find_arg = '-w'
+let s:tag_stack = []
 
 try
   if !has('win32') && $OSTYPE !~? 'cygwin\|linux' && system('uname -s') =~? 'SunOS' && system('uname -r') =~? '^5'
```

Comment:
> here it says we do not

Thanks, fixed. That was written in a time when libuv was younger.

> so then I should just change SunOS there to illumous

Sorry, I don't know. The least-risky thing is to do whatever Vim's man.vim does, unless there's an explicit reason not to.